### PR TITLE
update the Forgot root password

### PR DESCRIPTION
**Pull request by lifeijie6**
- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- 更新了‘忘记root密码’一栏的内容，使得操作更加清晰

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/commit/1344d306d7f920ec75b7fc5cd50239e19ae40f44

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tiancaiamao — APPROVED** [review]

File: user-account-management.md [hunk 1]
```diff
@@ -197,7 +197,7 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
 
 ## 忘记 `root` 密码
 
-1. 修改配置文件，在 `security` 部分添加 `skip-grant-table`：
+1. 修改配置文件，登录其中一台tidb-server所在机器，在tidb节点部署目录下的conf目录中，找到tidb.toml配置文件，在 `security` 部分添加 `skip-grant-table`，如无`security` 部分则将如下两行添加至tidb.toml配置文件尾部：
```
Comment (TomShawn) [inline user-account-management.md:200]:
```suggestion
1. 修改 TiDB 配置文件。

    1. 登录其中一台 tidb-server 实例所在的机器。
    2. 进入 TiDB 节点的部署目录下的 `conf` 目录，找到 `tidb.toml` 配置文件。
    3. 在配置文件的 `security` 部分添加配置项 `skip-grant-table`。如无 `security` 部分，则将以下两行内容添加至tidb.toml 配置文件尾部：
```

File: user-account-management.md [hunk 2]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
     skip-grant-table = true
     ```
 
-2. 使用修改之后的配置启动 TiDB，然后使用 `root` 登录后修改密码：
+2. 停掉tidb-server进程
```
Comment (TomShawn) [inline user-account-management.md:209]: 下面命令看起来终止的是 tikv-server？
```suggestion
2. 终止该 tidb-server 的进程：
```

File: user-account-management.md [hunk 3]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+
+    {{< copyable "shell-regular" >}}
```
Comment (TomShawn) [inline user-account-management.md:211]:
```suggestion
    查看 tikv-server 的进程：
```

File: user-account-management.md [hunk 4]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+
+    ```bash
+    ps aux |grep tikv-server
```
Comment (TomShawn) [inline user-account-management.md:214]:
```suggestion
    ps aux | grep tikv-server
```

File: user-account-management.md [hunk 5]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+    ```
+
+    找到tikv-server对应的pid并使用kill命令停掉进程
```
Comment (TomShawn) [inline user-account-management.md:217]:
```suggestion
    找到 tikv-server 对应的进程 ID (PID) 并使用 `kill` 命令停掉该进程：
```

File: user-account-management.md [hunk 6]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    kill -9 pid
```
Comment (TomShawn) [inline user-account-management.md:222]:
```suggestion
    kill -9 <pid>
```

File: user-account-management.md [hunk 7]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+    ```    
+
+3. 使用修改之后的配置启动 TiDB，在tidb节点部署目录下的scripts目录中，切换到操作系统`root`账号下在前台执行`run_tidb.sh`脚本，然后另开一个终端使用 `root` 登录后修改密码：
```
Comment (TomShawn) [inline user-account-management.md:225]:
```suggestion
3. 使用修改之后的配置启动 TiDB：

    1. 进入 TiDB 节点部署目录下的 `scripts` 目录，切换到操作系统 `root` 账号，在前台执行目录中的 `run_tidb.sh` 脚本，然后在新的终端窗口中使用 `root` 登录后修改密码：
```

File: user-account-management.md (hunk 3 above)
Comment (lifeijie6) [reply user-account-management.md:211]: 应该是 查看tidb-server的进程：

File: user-account-management.md [hunk 8]
```diff
@@ -206,14 +206,32 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
 
     {{< copyable "shell-regular" >}}
 
     ```bash
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
 
+4. 第3步中前台运行的`run_tidb.sh`脚本停掉，并去掉第1步中配置文件中添加的内容，等待确认tidb-server自启动
```
Comment (TomShawn) [inline user-account-management.md:233]:
```suggestion
4. 停止运行 `run_tidb.sh` 脚本，并去掉 第 1 步 中在 TiDB 配置文件中添加的内容，等待 tidb-server 自启动。
```

File: user-account-management.md (hunk 5 above)
Comment (lifeijie6) [reply user-account-management.md:217]: 找到 tidb-server 对应的进程 ID (PID) 并使用 `kill` 命令停掉该进程：

File: user-account-management.md (hunk 4 above)
Comment (lifeijie6) [reply user-account-management.md]: ps aux | grep tidb-server

File: user-account-management.md [hunk 9]
```diff
@@ -206,14 +210,37 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
     skip-grant-table = true
     ```
 
-2. 使用修改之后的配置启动 TiDB，然后使用 `root` 登录后修改密码：
+2. 终止该 tidb-server 的进程：
+
+    查看 tidb-server 的进程：
+
+    ```bash
+    ps aux | grep tikv-server
```
Comment (TomShawn) [inline user-account-management.md:218]:
```suggestion
    ps aux | grep tidb-server
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**